runtime/onert/backend/cpu/ShapeFixer.cc

```diff
@@ -64,17 +64,7 @@ void ShapeFixer::visit(const ir::operation::Sub &node)
   }
 }
 
-void ShapeFixer::visit(const ir::operation::Mul &node)
-{
-  // The same as Add
-  const auto lhs_index{node.getInputs().at(ir::operation::Mul::Input::LHS)};
-
-  // Quantization : not supported
-  if (_ctx.at(lhs_index).typeInfo().type() == ir::DataType::QUANT_UINT8_ASYMM)
-  {
-    throw std::runtime_error{"ShapeFixer: NYI for quantized Mul"};
-  }
-}
+void ShapeFixer::visit(const ir::operation::Mul &) { /* DO NOTHING */}
```

Comment:
You can remove this function if do nothing.

Reply:
removed visit() function.
Thanks for your comment : )